diff --git a/node_modules/shiki-languages/dist/lang.js b/node_modules/shiki-languages/dist/lang.js
index e02a2f4..84e15f3 100644
--- a/node_modules/shiki-languages/dist/lang.js
+++ b/node_modules/shiki-languages/dist/lang.js
@@ -62,6 +62,8 @@ exports.commonLangAliases = [
     'ts',
     'yml',
     'md',
+    'toml',
+    'undefined',
     /**
      * Extra grammars
      */
@@ -344,7 +346,7 @@ exports.languages = [
         id: 'shellscript',
         scopeName: 'source.shell',
         path: '../data/grammars/Shell-Unix-Bash.tmLanguage.json',
-        aliases: ['shell', 'bash', 'sh', 'zsh']
+        aliases: ['shell', 'bash', 'sh', 'zsh', 'undefined']
     },
     {
         id: 'sql',
@@ -392,7 +394,7 @@ exports.languages = [
         id: 'yaml',
         scopeName: 'source.yaml',
         path: '../data/grammars/yaml.json',
-        aliases: ['yml']
+        aliases: ['yml', 'toml']
     },
     /**
      * Extra grammars
