diff --git a/node_modules/@ssgjs/sapper/dist/export.js b/node_modules/@ssgjs/sapper/dist/export.js
index 6b1d535..d65224b 100644
--- a/node_modules/@ssgjs/sapper/dist/export.js
+++ b/node_modules/@ssgjs/sapper/dist/export.js
@@ -1859,16 +1859,28 @@ async function _export({
 		if (!message.__sapper__ || message.event !== 'file') return;
 		latestUrl = message.url;
 		const saveresult = save(message.url, message.status, message.type, message.body);
-		if (saveresult) saveresult.then(() => checkWritingDone(message.url));
+		// if (saveresult) saveresult.then(() => checkWritingDone(message.url));
+		checkWritingDone()
 	});
 
-	// swyx: introduced to solve race condition
-	// https://github.com/sveltejs/sapper/issues/893#issuecomment-538811020
-	function checkWritingDone(currentUrl) {
-		if (latestUrl != currentUrl || queue.getLength() > 0) ; else {
-			proc.kill();
-		}
-	}
+	var checkWritingDone = debounce(function() {
+		proc.kill();
+	}, 10000);
+
+	function debounce(func, wait, immediate) {
+		var timeout;
+		return function() {
+			var context = this, args = arguments;
+			var later = function() {
+				timeout = null;
+				if (!immediate) func.apply(context, args);
+			};
+			var callNow = immediate && !timeout;
+			clearTimeout(timeout);
+			timeout = setTimeout(later, wait);
+			if (callNow) func.apply(context, args);
+		};
+	};
 
 	function handle(url, fetchOpts, addCallback) {
 		let pathname = url.pathname;
